examples/tv-casting-app/linux/simple-app-helper.cpp

```diff
@@ -244,6 +246,22 @@ CHIP_ERROR InitCommissionableDataProvider(LinuxCommissionableDataProvider & prov
                          options.payload.discriminator.GetLongValue());
 }
 
+void PrintEndpointsVendorAndProductIDs(
```

Comment:
I think you could rename to something more succinct like the following:

```suggestion
void LogEndpoints(
```

Reply:
sure